src/alt-da/da-server/da_server_launcher.star

```diff
@@ -0,0 +1,96 @@
+shared_utils = import_module(
+    "github.com/ethpandaops/ethereum-package/src/shared_utils/shared_utils.star"
+)
+constants = import_module(
+    "github.com/ethpandaops/ethereum-package/src/package_io/constants.star"
+)
+
+# The dirpath of the data directory on the da-server container
+# note that we use /home which is available but not persistent
+# because we aren't mounting an external kurtosis file
+# this means that the data is lost when the container is deleted
+DATA_DIRPATH_ON_DA_SERVER_CONTAINER = "/home"
+
+# Port IDs
+DA_SERVER_HTTP_PORT_ID = "http"
+
+# Port nums
+DA_SERVER_HTTP_PORT_NUM = 3100
+
+
+def get_used_ports():
+    used_ports = {
+        DA_SERVER_HTTP_PORT_ID: shared_utils.new_port_spec(
+            DA_SERVER_HTTP_PORT_NUM,
+            shared_utils.TCP_PROTOCOL,
+            shared_utils.HTTP_APPLICATION_PROTOCOL,
+        ),
+    }
+    return used_ports
+
+def launch(
+    plan,
+    service_name,
+    da_server_extra_args,
+    generic_commitment,
+):
+
+    config = get_da_server_config(
+        plan,
+        service_name,
+        da_server_extra_args,
+        generic_commitment,
+    )
+
+    da_server_service = plan.add_service(service_name, config)
+
+    http_url = "http://{0}:{1}".format(da_server_service.ip_address, DA_SERVER_HTTP_PORT_NUM)
+    return new_da_server_context(
+        http_url=http_url,
+        generic_commitment=generic_commitment,
+    )
+
+
+def get_da_server_config(
+    plan,
+    service_name,
+    da_server_extra_args,
+    generic_commitment,
+):
+    image = "us-docker.pkg.dev/oplabs-tools-artifacts/images/da-server:devnet"
```

Comment:
the `devnet` tag doesn't exist and is overriding the `DEFAULT` in the input parser. In general I'd avoid overriding it this way. I'd also add the da_image as a config variable under `da_server_params` and then set the `image` return value for ServiceConfig as `image=da_server_params.da_server_image`. The input parser will populate the default value if none is set, so by default the `latest` image is used and if specified the config image is used.

Reply:
good catch! [4598487](https://github.com/ethpandaops/optimism-package/pull/69/commits/4598487706a2f87c6a06e0ed01e367bfa3dd79c4)